src/key.scad

```diff
@@ -286,7 +286,7 @@ module key(inset = false) {
   }
 
   // both stem and support are optional
-  if ($stem_type != "disable" || $stabilizer_type != "disable") {
+  if ($stem_type != "disable" || ($stabilizers != [] && $stabilizer_type != "disable")) {
```

Comment:
if $stabilizer_type != "disable" but $stabilizers == [], you would get the intersection render bug (where the envelope renders, since there's nothing to intersect it with). Checking if stabilizers is empty here fixes it. not sure if there's a better solution but also meh